docs/experimental-features/beaconing-detection.asciidoc

```diff
@@ -0,0 +1,50 @@
+[[beaconing-detection]]
+== Beaconing detection
+
+This feature provides an early warning system for command and control (C2) beaconing activity. It monitors network traffic for indicators of compromise (IOCs), and provides analytics to aid your threat hunting.
+
+[discrete]
+=== Deploy the Beaconing detection package
+
+To deploy Beaconing detection in your environment, follow https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/beaconing.md[these steps].
+
+The release package includes dashboards for monitoring beaconing activity in your environment. Users can review signals via a Lens dashboard called `Beaconing discovery`.
+
+[role="screenshot"]
+image::images/beaconing-detection-1.png[]
+
+[discrete]
+=== Feature details
+
+The feature uses a {ref}/transforms.html[transform] to segment network data by host and process name, then runs scripted metric aggregations on the host-process name pairs. For a given time window, the scripted metric aggregation checks each partition for the following:
+
+* Periodicity of signals, accounting for jitter.
```

Comment:
This seems pretty jargon-heavy. Some cybersecurity audiences might be familiar with "periodicity" and "jitter," but probably not general readers.

Reply:
I think that periodicity of signals = signals repeating at regular intervals, jitter = minor variations in those regular intervals. I feel like a user to look up the terms if they were unfamiliar, but we could also say "Signals repeating at regular intervals, accounting for minor variations in those intervals." Do you think that would be better @joepeeples

Reply:
@benironside Yes, that's a lot more readable to me, thanks!